modules/backend/classes/AuthManager.php

```diff
@@ -96,6 +96,26 @@ public function registerPermissions($owner, array $definitions)
         }
     }
 
+    /**
+     * Removes a single back-end permission
```

Comment:
@DanHarrin Would you mind adding the parameters and return value to the docblock?
```suggestion
     * Removes a single back-end permission
     * @param string $owner Specifies the menu item's owner plugin or module in the format Vendor/Module.
     * @param string $code The code of the permission to remove
     * @return void
```